djangocms_moderation/admin_actions.py

```diff
@@ -153,9 +154,11 @@ def add_items_to_collection(modeladmin, request, queryset):
         return HttpResponseRedirect(request.META.get("HTTP_REFERER"))
 
 
-add_items_to_collection.short_description = _(
-    "Add to moderation collection"
-)  # noqa: E305
+add_items_to_collection.short_description = _("Add to moderation collection")
+
+add_item_to_unpublish_collection = partial(add_items_to_collection)
+add_item_to_unpublish_collection.__name__ = 'add_item_to_unpublish_collection'
+add_item_to_unpublish_collection.short_description = _('Add moderation collection to unpublish')
```

Comment:
This is confusing: "Add moderation collection to unpublish", Do you mean Add items to a collection to unpublish?

Reply:
Yes. I'll change the wording on that one.

Reply:
@Aiky30 I changed the wording!